ijt-demo/requirements.txt

```diff
@@ -0,0 +1,3 @@
+asyncua==0.9.14
```

Comment:
with this version there might be some issues while importing the nodesets!
`-e git+https://github.com/FreeOpcUa/opcua-asyncio.git#egg=asyncua`
@swamper123

Reply:
since there are some big changes in asyncua (use of dataclasses instead of python classes) which are not released yet we are forced to use master till we made a release....

Reply:
@GoetzGoerisch can you take a look at the suggested changes? This has to do with your changes in the requirements.txt files

Reply:
Updated, sorry for the confusion!